openprescribing/frontend/tests/commands/test_import_measures.py

```diff
@@ -38,22 +39,45 @@ def test_measures():
     }
 
 
+def parse_args(*opts_args):
+        parser = argparse.ArgumentParser()
+        cmd = Command()
+        parser = cmd.create_parser("import_measures", "")
+        options = parser.parse_args(opts_args)
+        return cmd.parse_options(options.__dict__)
```

Comment:
This function is double-indented.

Also, why is the `.__dict__` necessary in `cmd.parse_options(options.__dict__)`?

Reply:
To the latter question: because that's what `call_command` does in django, and there's no subcommand we can call to replicate that.  Will add a comment.